src/components/molecules/FolderNavigatorItem.tsx

```diff
@@ -224,6 +239,81 @@ const FolderNavigatorItem = ({
     }
   }
 
+  const handleDragOverNavigatorItem = useCallback((e) => {
+    e.preventDefault()
+    e.stopPropagation()
+    e.dataTransfer.dropEffect = 'move'
+    setDraggedOver(true)
+  }, [])
+
+  const handleDragLeave = (e: React.DragEvent) => {
+    e.preventDefault()
+    e.stopPropagation()
+    console.log('Drag is over!')
+    setDraggedOver(false)
+  }
+
+  const handleDragStartNavigatorItem = (event: React.DragEvent) => {
+    setTransferableTextData(event, folderPathnameFormat, folderPathname)
+  }
+
+  const handleDropFolderToNavigatorItem = (
+    sourceItemPathname: string,
+    destinationFolderPathname: string
+  ) => {
+    const sourceFolderName = sourceItemPathname.substring(
+      sourceItemPathname.lastIndexOf('/')
+    )
+    const newFolderPathname = `${destinationFolderPathname}${sourceFolderName}`
+    renameFolder(
+      storageId,
+      sourceItemPathname == '' ? '/' : sourceItemPathname,
+      newFolderPathname
+    )
+      .then(() => {
+        // refresh works for file system (FSNote) database
```

Comment:
Dunno why, looked over code from Pouch and FSNote, seems similar, update is done on storage level,
But notes are not seen after dragging the folder into another place (it refreshes the folder location but sub-folders and notes disappear from view in Pouch DB. After `Ctrl+R` - reload, they are correctly placed in the new destination place.